CHANGELOG.md

```diff
@@ -5,8 +5,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0] - 2019-05-14
+
 * `[added]` `.clang-format` Style-file for clang-format and helper targets
             `make style-fix` and `make style-check` (#36)
+* `[changed]` Use stdint stype types (e.g. `uint16_t` instead of `u16`)
```

Comment:
I don't understand the wording with `stype` here. Shouldn't it just be `Use stdint types`

Reply:
yes, one for the "think before you type" category